### PR TITLE
Replace query builder setup icons with text

### PR DESCRIPTION
**Pull request by travisstebbins**
## Summary

Restyle the execution context panel in all query builder windows to replace symbols with text to make it easier for users to interpret the UI.

## How did you test this change?

- [x] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

Class query builder:
ClassQueryBuilder

Database query builder:
DatabaseQueryBuilder

Mapping execution query builder:
MappingExecutionQueryBuilder

Service query builder:
ServiceQueryBuilder

Basic query builder:
BasicQueryBuilder

### CODE REVIEW

File: packages/legend-query-builder/src/components/QueryBuilderSideBar.tsx [hunk 1]
```diff
@@ -156,257 +153,57 @@ export const QueryBuilderClassSelector = observer(
...
+            htmlFor="query-builder__setup__class-selector"
+          >
+            Entity
+          </label>
+          <CustomSelectorInput
+            inputId="query-builder__setup__class-selector"
+            className="panel__content__form__section__dropdown query-builder__setup__config-group__item__selector query-builder__setup__config-group__item__selector__milestoned"
+            placeholder={
+              classOptions.length
+                ? 'Choose a class...'
```
Comment (YannanGao-gs) [inline packages/legend-query-builder/src/components/QueryBuilderSideBar.tsx:169]: if we rename it to Entity, shall we change this too?

File: packages/legend-extension-dsl-data-space/src/components/query-builder/DataSpaceQueryBuilder.tsx [hunk 2]
```diff
@@ -217,147 +214,151 @@ const DataSpaceQueryBuilderSetupPanelContent = observer(
...
-                />
-              </div>
-            )}
+          )}
+          <div className="query-builder__setup__config-group__item">
+            <QueryBuilderClassSelector
+              queryBuilderState={queryBuilderState}
+              classes={classes}
+              onClassChange={queryBuilderState.onClassChange}
+              noMatchMessage="No compatible class found for specified execution context"
```
Comment (YannanGao-gs) [inline packages/legend-extension-dsl-data-space/src/components/query-builder/DataSpaceQueryBuilder.tsx:357]: do we need to rename from `class` to `entity` for consistency

**Comment by YannanGao-gs** [comment]
Personal css suggesstion 
Do you think if we should make the header `Properties` more similar to the`explorer tree` header
Screenshot 2024-05-15 at 5 17 55 PM

File: packages/legend-query-builder/src/components/QueryBuilderSideBar.tsx (hunk 1 above)
Comment (travisstebbins) [reply packages/legend-query-builder/src/components/QueryBuilderSideBar.tsx]: Done

File: packages/legend-extension-dsl-data-space/src/components/query-builder/DataSpaceQueryBuilder.tsx (hunk 2 above)
Comment (travisstebbins) [reply packages/legend-extension-dsl-data-space/src/components/query-builder/DataSpaceQueryBuilder.tsx]: Done

**Comment by travisstebbins** [comment]
> **Personal** css suggesstion Do you think if we should make the header `Properties` more similar to the`explorer tree` header Like the shaded box but keep the current size
> 
> Screenshot 2024-05-15 at 5 17 55 PM

Done

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**